interface/feature_selection.py

```diff
@@ -0,0 +1,60 @@
+import streamlit as st
+from .components import (
+    get_checkboxes,
+)
+from data_tools import separate_datasets
+
+
+def data_selection(data):
```

Comment:
Would language detection / adding derived features such as `AuthorProminence` also be a part of this step?